packages/repeater/src/bus/ExecuteRequestEventHandler.ts

```diff
@@ -0,0 +1,60 @@
+import { Protocol } from '../models';
+import { Request, RequestRunner, Response } from '../request-runner';
+import { bind, EventHandler } from '@secbox/core';
+import { injectable, injectAll } from 'tsyringe';
+
+interface ExecuteRequestPayload {
+  readonly protocol: Protocol;
+  readonly url: string;
+  readonly headers: Record<string, string | string[]>;
+  readonly method?: string;
+  readonly body?: string;
+  readonly correlation_id_regex?: string;
+}
+
+interface ExecuteRequestResult {
+  readonly protocol: Protocol;
+  readonly body?: string;
+  readonly headers?: Record<string, string | string[] | undefined>;
+  readonly status_code?: number;
+  readonly error_code?: string;
+  readonly message?: string;
+}
+
+@injectable()
+@bind('ExecuteScript')
+export class ExecuteRequestEventHandler
+  implements EventHandler<ExecuteRequestPayload, ExecuteRequestResult>
+{
+  constructor(
+    @injectAll(RequestRunner)
+    private readonly requestRunners: RequestRunner[]
+  ) {}
+
+  public async handle(
+    event: ExecuteRequestPayload
+  ): Promise<ExecuteRequestResult> {
+    const { protocol } = event;
+
+    const runner = this.requestRunners.find(x => x.protocol === protocol);
+
+    if (!runner) {
+      throw new Error(`Unsupported protocol "${protocol}"`);
+    }
+
+    const response: Response = await runner.run(
+      new Request({ ...event, correlationIdRegex: event.correlation_id_regex })
+    );
+
+    const { statusCode, message, errorCode, body, headers } = response;
+
+    return {
+      protocol,
+      body,
+      headers,
+      status_code: statusCode,
+      error_code: errorCode,
+      message
```

Comment:
The shorthand properties should be first (let's configure ESlint to verify such rules)
```suggestion
      protocol,
      body,
      headers,
      message,
      status_code: statusCode,
      error_code: errorCode
```

Reply:
> (let's configure ESlint to verify such rules)

https://github.com/fxOne/eslint-plugin-sort-keys-shorthand ?

Reply:
Ok, let's postpone that for some time